lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -56,6 +57,42 @@ window.runAudits = function(options, audits) {
       });
 };
 
+window.getListOfAudits = function() {
+  return _flatten(
+    configJSON.aggregations.map(aggregation => {
+      if (aggregation.items.length > 1) {
```

Comment:
is there are case now when we have just 1 item? 

also is it worth flipping the logic so we handle the case of `length === 1` at the top, whereas the default behavior is returning the items array? feels a bit more natural to me.. Not sure though.